_monitoring-plugins/alerting/security.md

```diff
@@ -31,6 +31,9 @@ Later, the user `psantos` wants to edit the monitor to run every two hours, but
 
 After making the change, the monitor now runs with the same permissions as `psantos`, including any [document-level security]({{site.url}}{{site.baseurl}}/security-plugin/access-control/document-level-security/) queries, [excluded fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-level-security/), and [masked fields]({{site.url}}{{site.baseurl}}/security-plugin/access-control/field-masking/). If you use an extraction query to define your monitor, use the **Run** button to ensure that the response includes the fields you need.
 
+Once a monitor is created, the Alerting plugin will continue executing the monitor, even if the user who created the monitor has permissions removed. To stop a monitor, a user with the correct cluster permissions must manually disable or delete the monitor. To disable a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/write`, and to delete a monitor, the user must have `cluster:admin/opendistro/alerting/monitor/delete`. This rule applies to all types of monitors.
+
+If your monitor's trigger has notifications configured, the Alerting plugin continues to send out notifications regardless of destination type. To stop notifications, a user must manually delete them in the trigger's actions.
```

Comment:
I have a voice question. Do we prefer to use "a user" or the less formal "you" when guiding the reader?

Reply:
We don't have a hard and fast rule about which one to use. In this case, I wanted to use "user" because of the association to roles and permissions, and I didn't want the reader to get confused about what they specifically can do about Alerting permissions.